examples/src/ContentExplorerModalContainerExamples.js

```diff
@@ -1,6 +1,6 @@
 import React, { Component } from 'react';
 
-import Button from '../../src/components/button';
+import Button from '../../src/components/button'; // eslint-disable-line
```

Comment:
A .js file can no longer import .ts file.
The compiled versions will be fine, but uncompiled this will be an issue, as such you will see this (and flow suppression of the same) in many files.

Reply:
would like more info where that does not work...

Reply:
Had to disable this eslint rule. Alternative is we add it to each line of import that causes the problem.